.github/check-format.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+source $(dirname "${BASH_SOURCE[0]}")/bot-base.sh
+
+echo -n "Collecting information on triggering PR"
+PR_URL=$(jq -r .pull_request.url "$GITHUB_EVENT_PATH")
```

Comment:
I guess you cannot use `-e` here because the result can be null?

Reply:
Exactly, usually I use -e to make the script fail fast if input is missing, i.e. `null` (also the reason I output dots between the steps), but in this situation I wanted to provide an alternative since `check-format` can be run both from a PR as well as a PR comment, which use different output formats.